src/org/joml/AABBi.java

```diff
@@ -0,0 +1,778 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2018-2020 JOML.
```

Comment:
Please change this to `Copyright (c) 2020 JOML.`
All source files' copyright begin with the year in which that file/class was introduced.